modules/renter/contractor/contractor_test.go

```diff
@@ -78,18 +78,6 @@ func TestNew(t *testing.T) {
 	if _, ok := err.(*json.SyntaxError); !ok {
 		t.Fatalf("expected invalid json, got %v", err)
 	}
-
-	// Corrupted logfile.
-	os.RemoveAll(filepath.Join(dir, "contractor.log"))
-	f, err := os.OpenFile(filepath.Join(dir, "contractor.log"), os.O_CREATE, 0000)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer f.Close()
-	_, err = New(stub, stub, stub, stub, dir)
-	if !os.IsPermission(err) {
-		t.Fatalf("expected permissions error, got %v", err)
-	}
```

Comment:
I couldn't find a way to create a file with bad permissions on Windows. On Linux I would use `Umask`, but there doesn't seem to be a Windows equivalent of that syscall.